checker/src/org/checkerframework/checker/oigj/OwnershipAnnotatedTypeFactory.java

```diff
@@ -47,10 +53,17 @@ public OwnershipAnnotatedTypeFactory(BaseTypeChecker checker) {
 
     @Override
     protected TreeAnnotator createTreeAnnotator() {
-        return new ListTreeAnnotator(
-                super.createTreeAnnotator(),
-                new OwnershipTreeAnnotator(this)
-        );
+        ImplicitsTreeAnnotator implicitsTreeAnnotator = new ImplicitsTreeAnnotator(this);
+        implicitsTreeAnnotator.addTreeKind(Tree.Kind.NEW_CLASS, MUTABLE);
+
+        implicitsTreeAnnotator.addTreeKind(Tree.Kind.CLASS, BOTTOM_QUAL);
+        implicitsTreeAnnotator.addTreeKind(Tree.Kind.ENUM, BOTTOM_QUAL);
+        implicitsTreeAnnotator.addTreeKind(Tree.Kind.ANNOTATION_TYPE, BOTTOM_QUAL);
+        implicitsTreeAnnotator.addTreeKind(Tree.Kind.NEW_ARRAY, BOTTOM_QUAL);
+
+
+        return new ListTreeAnnotator(implicitsTreeAnnotator, new PropagationTreeAnnotator(this),
```

Comment:
This is how other checkers add "implicits" procedurally, see UnitsAnnotatedTypeFactory for example.  The alternative is to call super.createTreeAnnotator, cast it to a ListTreeAnnotator, find the ImplicitTreeAnnotator -- that's not nice either.  Perhaps, we need another way to do this, but implemented that is orthogonal to this pull request.